net-im/discord/files/launcher.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/env bash
+# coding: UTF-8
+
+
+declare -a discord_parameters
+
+SECCOMP=false
+
+[[ ! "${SECCOMP}" ]] && discord_parameters+=( --disable-seccomp-filter-sandbox )
+
+[[ -n "${WAYLAND_DISPLAY}" ]] && discord_parameters+=(
+	--enable-features=UseOzonePlatform
+	--ozone-platform=wayland
```

Comment:
can we add --enable-wayland-ime

Reply:
Yes, no problem, done (note: I have no idea where to read about those options).

Reply:
me neither someone else told me about that one